Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -9,17 +9,14 @@
 import time
 import re
 
-if not demisto.getParam('proxy'):
-    del os.environ['HTTP_PROXY']
-    del os.environ['HTTPS_PROXY']
-    del os.environ['http_proxy']
-    del os.environ['https_proxy']
-
 # disable insecure warnings
 requests.packages.urllib3.disable_warnings()
```

Comment:
```suggestion
urllib3.disable_warnings()
```
Due to an upgrade of MyPy this will fix the build fails